doc/source/release-notes/Percona-Server-8.0.16-7.rst

```diff
@@ -0,0 +1,61 @@
+.. rn:: 8.0.16-7
+================================================================================
+|Percona Server| |release|
+================================================================================
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona
+Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0.
+|Percona Server| |release| is now the current GA release in the 8.0
+series. All of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features available in MySQL 8.0.16
+Community Edition
+<https://dev.mysql.com/doc/refman/8.0/en/mysql-nutshell.html>`__ in addition to
+enterprise-grade features developed by Percona.  For a list of highlighted
+@@ -35,7 +35,7 @@ features from both MySQL 8.0 and Percona Server for MySQL 8.0, please see the
+New Features
+================================================================================
+
+- `Key rotation for the redo log <https://www.percona.com/doc/percona-server/LATEST/management/data_at_rest_encryption.html>`__ has been redesigned to rotate the key with `SELECT rotate_system_key("percona_redo"). 
+
+Bugs Fixed
+================================================================================
+- The :variable: `buf_dblwr_flush_buffered_writes` must crash the server when an I/O error occurs to the parallel doublewrite buffer. The doublewrite buffer error returns to the caller. Bug fixed :psbug:`5678`.
```

Comment:
There is no "variable buf_dblw_flush_..."
"error returns to the caller" is mostly meaningless for the end user

Reply:
as @laurynas-biveinis mentioned there is not variable with this name (also there is a space between :variable: `'`

Reply:
Rewrote to: Parallel doublewrite buffer writes must crash the server on an I/O error occurs. Bug fixed :psbug:`5678`.